ethstorage/es-node
# sync -- make max request size can be set in the command line

Issue: https://github.com/ethstorage/es-node/issues/37
Make maxRequestSize can be set in the command line. It is used by p2p sync requests, to limit the response size returned by the server from the client size. `--p2p.max.request.size`  is added to set the value.

How to test: 
Run the node with the following command:
`--network devnet --datadir ./database --storage.files storage.dat --p2p.max.request.size 4194304 --l1.beacon http://65.108.236.27:5052  --l1.rpc http://65.108.236.27:8545 --storage.l1contract 0x882BC290fc22C330592819977c48968a62AE25f4 --storage.miner 0x5C935469C5592Aeeac3372e922d9bCEabDF8830d --l1.beacon-based-time 1695981612 --l1.beacon-based-slot 1 --p2p.listen.udp 30305 --log.level debug  --p2p.bootnodes enr:-Li4QPWz6DaLXwZ3vgUfx5i0M6ysMAhjlFrkLlzaKdvbMykrIzdncYXQG0rjF8anZfv65LpEuYVdOZPbjzwvthqUIq-GAYsoHnk0imV0aHN0b3JhZ2XbAYDY15SIK8KQ_CLDMFkoGZd8SJaKYq4l9MGAgmlkgnY0gmlwhEFs7BuJc2VjcDI1NmsxoQI2rSdv0CGNPia7uQgvYdt4XuW2Nkp062a0jsr-H-SJ4IN0Y3CCJAaDdWRwgnZh`

How to check:
Check the debug log like the following and check if the `count` value can be matched to `p2p.max.request.size` 
`Persisted set of kvs    count=32   bytes=****`
